packages/stt-adapters/gcp/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Converts GCP Speech to Text Json to DraftJs
+ * see `sample` folder for example of input and output as well as `example-usage.js`
+ */
+
+import generateEntitiesRanges from '../generate-entities-ranges/index.js';
+
+const NANO_SECOND = 1000000000;
+
+/**
+ * attribute for the sentences object containing the text. eg sentences ={ punct:'helo', ... }
+ *  or eg sentences ={ text:'hello', ... }
+ * @param sentences
+ */
+export const getBestAlternativeSentence = sentences => {
+  if (sentences.alternatives.length === 0) {
+    return sentences[0];
```

Comment:
I guess is fair to assume the first result in the alternatives is the most accurate? and not worth filter/find the one with the highest confidence score?
(in other STT services, generally that's the case and if it's not accurate the other alternatives are generally also not good, but not sure if this applies for google STT as well?)

Reply:
ah, sorry my bad, I just noticed you are actually doing a `reduce` to get the sentence with highest confidence score 👍